pravnik.csl

```diff
@@ -14,22 +14,34 @@
     <category citation-format="note"/>
     <category field="law"/>
     <issn>0231-6625</issn>
-    <summary>Czech ISO-690, full note, suitable for Pravnik - a journal published by the Institute of State and Law of the Czech Academy of Sciences.</summary>
-    <updated>2020-04-13T10:00:00+00:00</updated>
+    <summary>Suitable for Pravnik and its English version The Lawyer Quarterly - journals published by the Institute of State and Law of the Czech Academy of Sciences.</summary>
```

Comment:
@OTristanF, so Právník publishes in Czech, and The Lawyer Quarterly (https://tlq.ilaw.cas.cz/index.php/tlq) in English? If they're different journals, it would be better to keep this Právník style fixed to Czech, and create a "dependent" CSL style for The Lawyer Quarterly that points to the Právník style, but has its `default-locale` set to English. (see https://github.com/citation-style-language/styles/blob/master/dependent/nature-biotechnology.csl to see what a dependent style looks like)

Reply:
@rmzelle, thanks for the suggestion. I have created a dependent style.